README.md

```diff
@@ -7,12 +7,17 @@ Its basically an application making engine written in **C++**<br/>
 This can also be used an powerful **GameEngine**!<br/>
 
 ## :wrench: Project Setup :wrench:
-- This Project uses `Premake` as its build system (`CMake` will be added later)<br/>
+- This Project uses `Premake` as its build system (`CMake` support now available!)<br/>
 - Visual Studio 2019 is recommended in Windows<br/>
 - Start by cloning the repository with `git clone --recursive https://github.com/FahimFuad/LightEngine.git` to clone along with the submodules. If you have already cloned and you want to initialize the submodules, type `git submodule init` and to update the submodules type `git submodule update` in your project directory.
 - To genereate the project files for your IDE, select the actions of the premake and run them in the project directory. A list of actions of the premake can be found [here](https://github.com/premake/premake-core/wiki/Using-Premake)
 - Once premake generate the project files then you can hit the run button and make the engine up and running!<br/>
 
+## :wrench: CMake setup :wrench:
+ - Create a directory called in which to create your project files. (e.g. 'build')
```

Comment:
Create a directory called _**in**_ which to create your project files. (e.g. 'build'), is this line is correct? 😄

Reply:
This line can be: Create a directory in which to create your project files. (e.g. 'build') 🙂